subsys/bluetooth/audio/ascs.c

```diff
@@ -114,9 +138,29 @@ static void ase_status_changed(struct bt_bap_ep *ep, uint8_t old_state, uint8_t
 		bt_conn_get_info(conn, &conn_info);
 
 		if (conn_info.state == BT_CONN_STATE_CONNECTED) {
+			const uint8_t att_header_size = 3; /* opcode (1) + handle (2) */
```

Comment:
Magic numbers. The name of variable is inaccurate as it's notification header size.

Reply:
Not sure what to do about the magic numbers; the values are not in any public header file. The value is however described by the comment, right? 

I'd rather update this later if we ever add those values to a public header file, than introducing them as part of this. Alternatively we can implement a function `bt_gatt_max_ntf_size(conn)` to get the maximum notification size?